strongbox-web-core/src/main/java/org/carlspring/strongbox/rest/MetadataManagementRestlet.java

```diff
@@ -22,12 +16,21 @@
 import java.io.InputStream;
 import java.security.NoSuchAlgorithmException;
 
+import io.swagger.annotations.*;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Component;
+
 /**
  * @author Martin Todorov
  */
 @Component
 @Path("/metadata")
 @Api(value = "/metadata")
+@PreAuthorize("permitAll")
```

Comment:
@AlexOreshkevich ,
I think this needs to be furter fine-grained. Not everyone will be allowed to rebuild the metadata.